assembl/static2/js/app/components/cookiesBar.jsx

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { Link } from 'react-router';
+import { Button } from 'react-bootstrap';
+import classnames from 'classnames';
+import { getCookieItem, setCookieItem, getDiscussionSlug } from '../utils/globalFunctions';
+import { get } from '../utils/routeMap';
+
+type State = {
+  hide: ?boolean | string
+};
+
+class CookiesBar extends React.Component<*, State> {
```

Comment:
`type Props = {};` ?

Reply:
Allright, this is what I had done first but I thought it made no sense